.github/workflows/sync.yml

```diff
@@ -111,6 +111,9 @@ jobs:
     # lets us use actions like actions/checkout.
     # It also gives us a pretty job overview that makes it easy to spot issues with
     # particular tracks.
+    # This approach will stop working once there are >256 repos in the Exercism org.
+    # That's still far away, but once that happens, we can split up the workflow in several workflows,
+    # e.g. one for tracks, one for other repos.
```

Comment:
I just noticed we're actually above that already. Oops.

Reply:
So many repos!

Reply:
What would be the best approach?

Reply:
I think we can duplicate the workflow, and have one that exclusively spawns jobs for track repos, and the other that spawns jobs for non-track repos (we should be able to determine this via the Search API). Then keep the rest of the workflow identical for simplicity.

Reply:
👍 Sounds good